net/ServerSocket.hpp

```diff
@@ -121,6 +118,8 @@ class ServerSocket : public Socket
         return _sockFactory->create(fd, type);
     }
 
+    static constexpr const char * ErrorMessagePrefix = "Failed to accept. (errno: ";
```

Comment:
Why pull this far away from the message we create ? there can be no significant performance win - but there is a real readability hit from having that far from the code that it is used in. Please remove.

Reply:
just the idea from earlier code and reusage .. but sure, making it local